abm/server.py

```diff
@@ -175,8 +180,12 @@ def do_put(self, context, descriptor, reader, writer):
             record_reader = reader.to_reader()
             while True:
                 try:
+                  # convert from Arrow format to Airbyte format
                   batch = record_reader.read_next_batch()
-                  connector.write_dataset_bytes(socket, batch.to_pandas().to_json(orient='records', lines=True).encode())
+                  df = batch.to_pandas()
+                  for _, row in df.iterrows():
+                    record_message=AirbyteMessage(type=MessageType.RECORD, record=AirbyteRecordMessage(stream=stream_name, data=row, emitted_at=111)).json()
+                    connector.write_dataset_bytes(socket,record_message.encode('utf-8'))
```

Comment:
What is the meaning of the value 111?

Reply:
its a dummy value. I will change that to take the real time. Thanks